__tests__/options.test.ts

```diff
@@ -76,13 +78,27 @@ describe('options', () => {
   });
 
   it('normalize monitor configs', () => {
+    const config = join(__dirname, 'fixtures', 'synthetics.config.ts');
+    expect(normalizeOptions({ config }, 'push')).toMatchObject({
+      screenshots: 'off',
+      schedule: 10,
+      privateLocations: ['test-location'],
+      locations: ['us_east'],
+    });
+
     expect(
-      normalizeOptions({
-        schedule: 3,
-        privateLocations: ['test'],
-        locations: ['australia_east'],
-      })
+      normalizeOptions(
+        {
+          config,
+          schedule: 3,
+          screenshots: 'only-on-failure',
+          locations: ['australia_east'],
+          privateLocations: ['test'],
+        },
+        'push'
```

Comment:
I'm seeing this in my vscode:

![Uploading Screenshot 2023-06-12 at 11.30.12.png…]()